### DISCUSSION TITLE
arco.design & semi.design

### DISCUSSION THREAD

**Discussion by myesn** [DouyinFE/semi-design#140] [Q&A]
Both [arco.design](https://arco.design) and [semi.design](https://semi.design) are ByteDance companies, which one should we use? 

[arco.design](https://arco.design) is more comprehensive, isn't it? 

It's hard to choose. They're all pretty good UI libraries. 🤔

**Comment by wuhw** [comment]
Both Semi and Arco are component libraries that have be used for more than 2 years in bytedance, and both have certain guarantees in terms of stability.
We suggest that you consider the following aspects:
1. Business scenario: vue, react, mobile?
2. UI style & custom theme
3. The experience of using complex and commonly used components: Form, Table and so on
